pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -54,6 +54,9 @@ const (
 	gceAffinityTypeClientIP = "CLIENT_IP"
 	// AffinityTypeClientIPProto - affinity based on Client IP and port.
 	gceAffinityTypeClientIPProto = "CLIENT_IP_PROTO"
+
+	operationPollInterval        = 3 * time.Second
+	operationPollTimeoutDuration = 7 * time.Minute
```

Comment:
It's generally a violation of the GCE api contract for an operation to never go to DONE, but there are known edge cases where it can take 15-20 minutes or even more to get there.

Does the pd retry code handle the case where there's an already-pending operation to detach a disk and GCE returns an error creating a new operation to do the same thing?  If not, then retries may not be improving reliability.